sdk/identity/azure-identity/src/test/java/com/azure/identity/implementation/HttpPipelineAdapterTests.java

```diff
@@ -24,38 +20,36 @@
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Mockito.when;
 
-@RunWith(PowerMockRunner.class)
-@PowerMockIgnore({"com.sun.org.apache.xerces.*", "javax.xml.*", "org.xml.*"})
-@PrepareForTest({HttpPipeline.class, HttpRequest.class, HttpResponse.class, URL.class})
 public class HttpPipelineAdapterTests {
 
     @Test
     public void testSendRequest() throws Exception {
         Mono<String> bodyResponse = Mono.just("dummy-body");
-        HttpPipeline pipeline = PowerMockito.mock(HttpPipeline.class);
+        HttpPipeline pipeline = Mockito.mock(HttpPipeline.class);
```

Comment:
nit: `HttpPipeline` and `HttpRequest` don't need to be mocked as they're able to be constructed in a mocked style

Reply:
This is part of the "mechanical translation" I was talking about. I think another pass on these is totally reasonable at some point.